packs/lastline/actions/lib/analysis_apiclient.py

```diff
@@ -0,0 +1,1890 @@
+#!/usr/bin/python
```

Comment:
My editor will crash loading this single humongous file. Break it down, break it down la la la la.